hack/validate-generated-config.sh

```diff
@@ -14,20 +14,88 @@ ci_operator_dir="$( dirname "${BASH_SOURCE[0]}" )/../ci-operator"
 
 cp -r "${ci_operator_dir}" "${workdir}"
 
-"$( dirname "${BASH_SOURCE[0]}" )/update-generated-config.sh"
+ci-operator-prowgen --from-dir "${ci_operator_dir}/config" --to-dir "${workdir}/ci-operator/jobs"
 
-if ! diff -Naupr "${ci_operator_dir}" "${workdir}/ci-operator"> "${workdir}/diff"; then
-  cat << EOF
+blacklist=(
```

Comment:
Nit: it may seem like a small thing, but in the future please use allowlist/blocklist. It's clearer and has less baggage.